### PR TITLE
Update dlp-conditions-and-exceptions.md

### PR DESCRIPTION
**Pull request by Pulkit-Sachan**

### CODE REVIEW

File: microsoft-365/compliance/dlp-conditions-and-exceptions.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ To configure the sender address location at a DLP rule level, the parameter is *
...
 |Document name contains words|condition: *DocumentNameMatchesWords* <br/><br/> exception: *ExceptIfDocumentNameMatchesWords*|Words|Messages where an attachment's file name matches any of the specified words.|
-|Document name matches patterns|condition: *DocumentNameMatchesPatterns* <br/><br/> exception: *ExceptIfDocumentNameMatchesPatterns*|Patterns|Messages where an attachment's file name contains text patterns that match the specified regular expressions.|
+|Document name matches patterns|condition: *DocumentNameMatchesPatterns* <br/><br/> exception: *ExceptIfDocumentNameMatchesPatterns*|Patterns|Messages where an attachment's file name contains text patterns that match the specified regular expressions (Discontinued for SharePoint and OneDrive workloads. Existing rules can't be modified and new rules can't be created. However, existing customers can continue to use the predicate).|
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-conditions-and-exceptions.md]:
```suggestion
|Document name matches patterns|condition: *DocumentNameMatchesPatterns* <br/><br/> exception: *ExceptIfDocumentNameMatchesPatterns*|Patterns|Messages where an attachment's file name contains text patterns that match the specified regular expressions. (Discontinued for SharePoint and OneDrive workloads. Existing rules can't be modified and new rules can't be created. However, existing customers can continue to use the predicate.)|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Pulkit-Sachan, Kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@Pulkit-Sachan

**Comment by yogkumgit** [comment]
@Pulkit-Sachan Gentle reminder.

**Pull request merged**